notebooks/evaluation/rag_baseline_evaluation.ipynb

```diff
@@ -0,0 +1,3387 @@
+               "output_type": "stream",
+               "text": [
+                  "eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX3V1aWQiOiJhYWFhYWFhYS1hYWFhLWFhYWEtYWFhYS1hYWFhYWFhYWFhYWEifQ.kwzm-8i8SveeqYqvsRUm4FiB7nd3I43aI70ImljgdudKM4xrDw9z3CUpEBRwqqh6D3ZghB2T-Lu7BlV36VR5sg\n"
+               ]
+            }
+         ],
+         "source": [
+            "bearer_token = jwt.encode({\"user_uuid\": str(UUID(\"aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa\"))}, key=\"your-secret-key\", algorithm=\"HS512\")\n",
+            "print(bearer_token)"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "#### First need to upload files that we are going to 'RAG with'\n",
+            "\n",
+            "If files have already been uploaded you can skip this step"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "#### Get files that correspond to the version of evaluation dataset  <a class=\"anchor\" id=\"files\"></a>\n",
+            "\n",
+            "Copy all the files that match your {DATA_VERSION} into `notebooks/evaluation/data/{DATA_VERSION}/raw/`. Find these files on the shared Google Drive and the corresponding version number/location\n",
+            "\n",
+            "**It is really important to use the same files that were used to genearte this particular version of the evaluation dataset. A mismatch between the two will result in inaccurate evaluatoin metrics**"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "**Only if you haven't uploaded files already** uncomment and run cell below"
+         ]
+      },
+      {
+         "cell_type": "code",
+         "execution_count": 13,
+         "metadata": {},
+         "outputs": [],
+         "source": [
+            "# url = 'http://127.0.0.1:5002/file/upload'\n",
+            "\n",
+            "# headers={\n",
+            "#     'accept': 'application/json',\n",
+            "#     \"Authorization\": f\"Bearer {bearer_token}\"\n",
+            "# }\n",
+            "# for file in V_RAW.glob(\"*.*\"):\n",
+            "#     files = {'file': open(file, 'rb')}\n",
+            "#     upload_file_response = requests.post(url, headers=headers, files=files)\n",
+            "\n",
+            "#     #TODO: Add some login in the loop to deal with status codes != 200\n",
+            "#     # if upload_file_response.status_code != 200:\n",
+            "#     #     print(\"Failed to upload data:\", upload_file_response.status_code)"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "List files uploaded to server & view JSON response"
+         ]
+      },
+      {
+         "cell_type": "code",
+         "execution_count": 14,
+         "metadata": {},
+         "outputs": [
+            {
```

Comment:
Should we be clearing outputs so it's clearer for the user what needs to be done?

Reply:
I am torn on this one. Usually I would 100% clear outputs, however, as there is a cost associated with running the evals I left them in. However, the long eval outputs are a bit unruly! Perhaps we could take a screen shot of what the end result is and pop it in as markdown cell if we _really_ want to show what the output looks like. Thoughs?